neuromation/api/jobs.py

```diff
@@ -41,10 +41,14 @@
     _raise_on_error_chunk,
     _try_parse_image_progress_step,
 )
-from .parser import Parser, SecretFile, Volume
+from .parser import DiskVolume, Parser, SecretFile, Volume
```

Comment:
Please replace `DiskVolume` -> `Disk`, `disk_volumes_` -> `disks` etc. here and below.
Shorter is better.

Reply:
I'm not sure how to do this properly. In another PR(https://github.com/neuromation/platform-client-python/pull/1716) I  add another `Disk` dataclass, so I cannot rename this one. Also in my mind the disk and disk volume is two different concepts (same way as **Secret** and **SecretFile**):
- **Disk** is a storage device, it has some capacity, status, owner, and lifespan in the future. It can exist without any container.
- **DiskVolume** is just specification on how to mount some **Disk** to some container.

On the other side, I am not sure that naming is correct. We can use something like **DiskMountSpec**, but it even longer.

Also currently `DiskVolume` and `Volume` dataclasses have no real difference. We can reuse `Volume` for disk mounts, and make API code separate them just before sending it to the server. For user code, there will be no difference between mounting `storage://` or `disk://`. But this is somehow implicit logic as for me and I prefer to avoid such a solution.

Reply:
ok, got it